docs/src/piccolo/getting_started/setup_mysql.rst

```diff
@@ -0,0 +1,28 @@
+.. _setting_up_mysql:
+
+###########
+Setup MySQL
+###########
+
+Installation
+************
+
+Follow the  `instructions for your OS <https://dev.mysql.com/doc/refman/8.4/en/installing.html>`_.
+
+
+Creating a database
+*******************
+
+Using ``mysql``:
+
+.. code-block:: bash
+
+    mysql -u root -p
+
+Enter the your password and create database:
```

Comment:
Can we change this to:

`Enter your password and create the database:`

Reply:
Thanks. I'm not a native English speaker and these mistakes always creep in. I would change that.